test/conversations_page_test.dart

```diff
@@ -40,13 +41,13 @@ void main() {
 
         await tester.pumpAndSettle();
 
-        expect(find.text('Leon, Noel'), findsOneWidget);
```

Comment:
I like the idea of leaving this here as well as the next line. Easier for the developer know exactly how it behaves without needing to dive in. Keep the following line too.

Reply:
Thanks for the suggestion. I don't agree that hardcoding the string makes it easier for the developer. I think it reduces clarity. Happy to discuss further. Given that you approved the PR I'll push it through to keep us moving but happy to discuss further on the discord or messenger or wherever.

On a meta note: I would appreciate suggestions in reviews coming in the form of requests rather than commands. Please feel free to pull me up on that if I'm not doing it. I've added a note to the wiki on this, please feel free to edit:
https://github.com/Adventures-In/chat_app/wiki/Contributing#reviewing-a-pr